sky/jobs/server/core.py

```diff
@@ -576,7 +576,99 @@ def _maybe_restart_controller(
 
 
 @usage_lib.entrypoint
-def queue(
+def queue(refresh: bool,
```

Comment:
Do we have to create a second function in the backend? Can we just do it on the API level?

Reply:
Also, do we even need to create a v2 version? Can we just make it work with RequestBody level compatibility, as our backward compatibility guide doc suggested?

Reply:
We can have a unified `queue()` impl, with different encoding at `v1` and `v2` APIs